### ISSUE TITLE
Videos: 

### ISSUE THREAD

**Issue by TripleCxx** [duckduckgo/zeroclickinfo-spice#2985]
https://github.com/duckduckg
https://github.com/duckduckgo/zeroclickinfo-spiceo   

------
IA Page:  http://duck.co/ia/view/videos 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @bsstoner

**Comment by GuiltyDolphin** [comment]
@TripleCxx Could you expand this comment to make it a bit less... Cryptic?

Are you experiencing any issues using Videos, have you noticed a bug, or perhaps you have a feature suggestion?

**Comment by TripleCxx** [comment]
Sry I've been busy with the day job, yes I'm experiencing bugs but prob not the same as yours? I have 2 GPUs with a sli connect, this setup was confused  for securing the back end of a server, Long agonising story,  I was successful in getting the local account confined normally but am running into obstacles, I'll be home in a lil while and will pick it back up, sorry for not getter back w ya 
Get Outlook for iOS

Get Outlook for iOS




On Mon, Aug 29, 2016 at 4:22 PM -0500, "Ben Moon" <notifications@github.com> wrote:












@TripleCxx Could you expand this comment to make it a bit less... Cryptic?



Are you experiencing any issues using Videos, have you noticed a bug, or perhaps you have a feature suggestion?

**Comment by TripleCxx** [comment]
sorry for the confusion, to many things going at once, the result query output from duckduckgo.  I also used their browser quiet a bit for privacy so i wanted to reference the search engine. I am not experiencing crypto problems, although it would not be hard to mis config your network with such a strong encryption. I built on to this in order to fix a system that had been victim to malware via ISO and Android  bots, malware, and brute force attacks so had to use some imagination.. lol I do have some kink i need to debug
